src/api.ts

```diff
@@ -15,7 +14,7 @@ function now() {
   );
 }
 
-export const createNewCustomer = async (accountId: string) => {
+export const createNewCustomer = async (accountId: string, API_BASE_URL: string) => {
```

Comment:
arg should not be formatted like a constant, also might be nice to just set the default value of this arg to `app.papercups.io` or whatever our hosted url will be